core/templates/dev/head/pages/base.html

```diff
@@ -207,6 +207,7 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/alertsService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/explorationContextService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/utilsService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/RTEHelperService.js"></script>
```

Comment:
Same comment as in the other PR -- please analyze where the RTE helper service is used, and import it in those locations only.

Also, could we call this RteHelperService (in line with HtmlEscaperService and IdGenerationService), so that only the first letter of the abbreviation is capitalized? I realize it was wrong in the original description of #3826, sorry -- I'll change that.